spec/features/search/advanced_search/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe "As a visitor" do
```

Comment:
New test verifying that what is expected appears on the advanced search page. Not currently clicking buttons or filling in anything.